### DISCUSSION TITLE
Alternate Post Section

### DISCUSSION THREAD

**Discussion by imagelessthought** [cotes2020/jekyll-theme-chirpy#1230] [Q&A]
I wanted to create a separate section of posts, one for general-public posts and another for posts only of interest to the group I work with.  

I created an index2.html, homealt.html, and defaultalt.hmtl that were exact copies of the originals. I know I will need to creaet alt version of post, tag, tags, category, and categories as well but nothing worked as epxted so I have not gotten there yet. The idea is each index would just show different "posts" that were unrelated to one another ("articles": different tags and categories as well and such). 

I updated index2 to use the homealt layout and homealt uses defaultalt as the layout.  Since everything is still identical to the orignals I thought the output would be identical at this point (still showing posts) and I would continue from there.

It did not work:

Right now everything is identical and still pointing to the default posts.
The second index does not get the css.
The second index does not show anything but the first post and part of the second.

There are issues with second set of posts ("_articles") as well: they are not beiong converted to html and articles folder is not created in _site. 

Any ideas?  I am at the initial stages of setting this part up but nothing is working as hoped.

**Comment by imagelessthought** [comment]
https://github.com/imagelessthought/jekyll-chirpy/tree/problems (not the master branch)

Everything is here if you want to take a look.

**Comment by imagelessthought** [comment]
Found a partial solution. I can hide posts from the index:

1. Change line 38 of home.html 
    from: {% assign posts = posts | push: default[i] %}
    to: {% assign posts = posts | push: default[i] | **where: 'search', 'include'** %}
2. Add this line to the front matter of the posts you want to be shown:
  search: include

**search: exclude** is supposed to remove the post from search engines.  It may do that but it still shows u p in the search results from the searches within Chirpy.

But they still show up.  Any suggestions on removing some posts from the search results from within Chirpy?

**Comment by imagelessthought** [comment]
I can rmeove the tags and categories from the posts and so the do not show up on those tabs, but they also show up in archives - which I would like to rmeove as well.

**Comment by imagelessthought** [comment]
Found alternative solution.

**Reply by Parad0x84** [reply]
Can you share it with me? I'm trying to create something similar but for portfolio purposes.
I want portfolio posts to not show up on any place except the tab I created for it.

Note: I will probably change my website in meantime, so I'm leaving a screenshot
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/36677584/19d06bad-da6b-4780-ad27-a3bcf755eee3)


Edit:
Site: https://paradox84.dev/
Repo: https://github.com/Parad0x84/Parad0x84.github.io

**Reply by imagelessthought** [reply]
I have not worked on it in a while.  I didn't really find a satisfactory solution.  But here is the repo I worked on:

https://github.com/imagelessthought/imageless-chirpy

**Reply by Parad0x84** [reply]
I think I got it.
I will explain it here soon: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1402#discussion-5916937

Thanks for help tho :)
